stubs/pyinstaller/PyInstaller/building/api.pyi

```diff
@@ -118,17 +123,19 @@ class EXE(Target):
         exclude_binaries: bool = False,
         bootloader_ignore_signals: bool = False,
         console: bool = True,
+        hide_console: _HideConsole = None,
         disable_windowed_traceback: bool = False,
         debug: bool = False,
         name: str | None = None,
         icon: _IconParam = None,
         version: _VersionParam = None,
         manifest: _ManifestParam = None,
-        embed_manifest: bool = True,
+        embed_manifest: Literal[True] = True,
```

Comment:
Deprecated in 6.0, will raise with a non-truthy value

Reply:
https://github.com/pyinstaller/pyinstaller/blob/dd40fb2cd75b86b5bbe23843bafde09740f451af/PyInstaller/building/api.py#L480-L484 reminds me very strongly of https://quotepark.com/quotes/1458843-douglas-adams-arthur-dent-what-happens-if-i-press-this-button/...